samples/bot-formatting-cards/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-formatting-cards-nodejs",
+    "source": "officeDev",
+    "title": "Different Formatting On Cards",
+    "shortDescription": "Demonstrating the feature where user can send different formatting on cards.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-formatting-cards/nodejs",
+    "longDescription": [
+      "Demonstrating the feature where user can send different formatting on cards using bot."
```

Comment:
same for here also "Demonstrating the feature where user can send different formatting on adaptive cards using bot."

Reply:
Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/689/files#diff-824fbd3defb344889effcfecba9c5a1ae303b488e565c47c32494597545bba6b

Reply:
Fixed